src/main/java/com/wavefront/sdk/grpc/WavefrontServerTracerFactory.java

```diff
@@ -111,24 +126,25 @@ private Span createServerSpan(Metadata headers, String methodName) {
     if (tracer == null) {
       return null;
     }
-    Map<String, String> headerMap = new HashMap<String, String>();
+    Map<String, String> headerMap = new HashMap<>();
     for (String key : headers.keys()) {
       if (!key.endsWith(Metadata.BINARY_HEADER_SUFFIX)) {
         String value = headers.get(Metadata.Key.of(key, Metadata.ASCII_STRING_MARSHALLER));
         headerMap.put(key, value);
       }
     }
     Span span;
+    String spanName = spanNameOverride != null ? spanNameOverride.apply(methodName) : methodName;
```

Comment:
It seems by default it's still using methodName if spanNameOverride is not provided?

Reply:
yes. FullMethodName will be default.